### ISSUE TITLE
Amp Validation Error: The attribute 'animate' in tag 'amp-accordion' is set to the invalid value 'true'.

### ISSUE THREAD

**Issue by atishjain29100** [ampproject/amphtml#24536]
I have used 'animate' attribute on amp-accordion tag. It works fine but the amp validation is failing as it says:
The attribute 'animate' in tag 'amp-accordion' is set to the invalid value 'true'.

I am not able to identify why amp-validation error is arising. Please provide a fix for this

**Comment by cathyxz** [comment]
The `animate` attribute isn't supposed to take a value. It should just be used as: 
```
<amp-accordion ... animate ...> 
// Contents 
</amp-accordion> 
```
Feel free to take a look at this example: https://playground.amp.dev/?url=https%3A%2F%2Fpreview.amp.dev%2Fdocumentation%2Fexamples%2Fcomponents%2Famp-accordion%2F21&format=websites&_gl=1*10kg7to*_ga*YW1wLWdqU2tnd0ZPZFFEbERPUnQ0cUlONXc.

**Comment by atishjain29100** [comment]
Hi,

I used the attribute in the same way in my code and it works as it is supposed to with smooth transition.

But the amp-validator fails on my page and it says "The attribute 'animate' in tag 'amp-accordion' is set to the invalid value 'true'". Not able to understand this behaviour. Please help regarding this.

**Comment by honeybadgerdontcare** [comment]
Have you looked at the HTML that is being returned from your server? Perhaps some tooling is adding the `=true` instead of preserving the attribute as just `animate`.

**Comment by atishjain29100** [comment]
Thanks !! This helped. Passed the attribute as animate = "" and it works like a charm :)
